packages/flutter/lib/src/material/ink_ripple.dart

```diff
@@ -31,19 +31,10 @@ RectCallback _getClipCallback(RenderBox referenceBox, bool containedInkWell, Rec
 }
 
 double _getTargetRadius(RenderBox referenceBox, bool containedInkWell, RectCallback rectCallback, Offset position) {
-  if (containedInkWell) {
-    final Size size = rectCallback != null ? rectCallback().size : referenceBox.size;
-    return _getRippleRadiusForPositionInSize(size, position);
-  }
-  return Material.defaultSplashRadius;
-}
-
-double _getRippleRadiusForPositionInSize(Size bounds, Offset position) {
-  final double d1 = (position - bounds.topLeft(Offset.zero)).distance;
-  final double d2 = (position - bounds.topRight(Offset.zero)).distance;
-  final double d3 = (position - bounds.bottomLeft(Offset.zero)).distance;
-  final double d4 = (position - bounds.bottomRight(Offset.zero)).distance;
-  return math.max(math.max(d1, d2), math.max(d3, d4)).ceilToDouble();
+  final Size size = rectCallback != null ? rectCallback().size : referenceBox.size;
```

Comment:
@HansMuller Hi,
I found that this processing is different from `InkSplash._getTargetRadius`, but they are the same before this PR.
 I want to know why you do this change? It seems not relevant to the description of this PR.
After the change, in some cases, the ripple cannot cover the entire row such as #75862.

CC/ @shihaohong

Reply:
Hmm, I have no context for this change either. @HansMuller might be able to give more details when he's more available (he's the Flutter Gardener for the week), but my only guess could be that the spec had updated at the time and that `InkSplash._getTargetRadius` did not receive the update?

Reply:
Ping @HansMuller Hi, Please take a look at this if you have time, also #75862 #76057

Reply:
@HansMuller Hi, any news for this? 😄

Reply:
Ping @HansMuller :)

Reply:
@HansMuller 😄